doc/controls/ExtendedSplashScreen.md

```diff
@@ -3,30 +3,60 @@ uid: Toolkit.Controls.ExtendedSplashScreen
+
+> [!TIP]
+> Users expect applications to load quickly after they launch them, so it is important to minimize the duration that a splash screen is displayed. `ExtendedSplashScreen` is intended to support a consistent user experience across platforms where scenarios necessitate a longer startup process.
 
 ## Properties
 
-| Property          | Type                   | Description                                                                                                                                                                                                                                          |
-|-------------------|------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `Platforms`       | `SplashScreenPlatform` | Gets or sets which platforms load the native splash screen (`All` by default). This is an flag enumeration, meaning you can specify multiple values that will be combined (eg `Android,iOS` will show native splash screen for only Android and iOS) |
-| `SplashScreen`    | `SplashScreen`         | Gets or sets the `SplashScreen` entity that's available for UWP that defines where the splash screen image is positioned.                                                                                                                            |
-| `Window`          | `Window`               | Gets or sets the `Window` for the application that is used to size the splash screen.                                                                                                                                                                |
-| `SplashIsEnabled` | `bool`                 | Gets a value that determines whether native splash is to be displayed for the current platform at runtime. Note for Android 12+ the native splash screen is always disabled irrespective of the Platforms value.                                     |
+| Property | Type | Description | Value |
+|----------|------|-------------|---------------|
+| **Platforms** | `SplashScreenPlatform` | Gets or sets the platform(s) where extended splash screen should be used. This is a flag enumeration, which allows for combining multiple values. | Default value is **All**. Other possible values include **Android**, **iOS**, **Windows**, **WebAssembly**, **Skia**, and **None**. |
+| **SplashIsEnabled** | `bool` | Gets a value representing whether the current environment is to display this splash screen. | **True** if the current platform is included in the **Platforms** property, otherwise **false**. |
+<!-- | **Window** | `Window` | Gets or sets a window for the application that is used to size the splash screen. | -->
```

Comment:
This one is commented, why?

Reply:
@agneszitte @kazo0 The Window property does not appear to be set or used very often internally, so I was confused as to whether we should document it.

Reply:
@nickrandolph Is the Window property needed?